.github/workflows/pkgdown.yaml

```diff
@@ -219,7 +222,7 @@ jobs:
           ref: "gh-pages"
 
       - name: Create and publish docs ↗️
-        uses: insightsengineering/r-pkgdown-multiversion@v2
+        uses: insightsengineering/r-pkgdown-multiversion@publish-reports-to-gh-pages
```

Comment:
Update after merging https://github.com/insightsengineering/r-pkgdown-multiversion/pull/11

Reply:
No changes to `r-pkgdown-multiversion` expected for now.